restore-sql-server-transaction-logs/scheduled-upload/upload-script.ps1

```diff
@@ -0,0 +1,46 @@
+$settingsFile="<full-path-to-your-script>\settings.json"
+$uploadedObjectsNumber=0
+$currentDateTimeUtc = (Get-Date).ToUniversalTime()
+
+# read from json settings file
```

Comment:
When you write comments, focus on explaining the why, not the what.

Reply:
Done, thanks.

Reply:
you can resolve this :)